### PR TITLE
added scheme as an option for rabbitmq.coffee

### PR DESCRIPTION
**Pull request by davidsulpy**
This would allow the rabbitmq script to use https where as before was hardcoded for http.

### CODE REVIEW

File: src/scripts/rabbitmq.coffee [hunk 1]
```diff
@@ -21,10 +21,9 @@
 # Notes:
 #
 # Author:
-#   kevwil
+#   kevwil, davidsulpy
 
-
-host = process.env.HUBOT_RABBITMQ_HOST
```
Comment (technicalpickles) [inline src/scripts/rabbitmq.coffee]: Looks like a mismatch here between what's documented to use (HUBOT_RABBITMQ_ROOT_URL) and what is used from the code (HUBOT_RABBITMQ_HOST)

Comment (davidsulpy) [inline src/scripts/rabbitmq.coffee]: Good catch, exactly right. I've updated this.

**Comment by technicalpickles** [comment]
@davidsulpy sorry for the delay here, but we've added a CLA since you first wrote this. You should be able to accept it at https://cla.github.com/github/hubot-scripts/accept/davidsulpy and then I can merge.

**Comment by davidsulpy** [comment]
Accepted. Thanks.

**Pull request merged**